flake.nix

```diff
@@ -2,26 +2,30 @@
 {
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs";
-    flake-utils.url = "github:numtide/flake-utils";
+
+    # <https://github.com/nix-systems/nix-systems>
+    systems.url = "github:nix-systems/default";
   };
 
-  outputs = {
-    self,
-    nixpkgs,
-    flake-utils,
-  }:
-    flake-utils.lib.eachDefaultSystem (system: let
-      pkgs = nixpkgs.legacyPackages.${system};
-      libselinuxPath = with pkgs;
-        lib.makeLibraryPath [
-          libselinux
-        ];
-      libaclPath = with pkgs;
-        lib.makeLibraryPath [
-          acl
-        ];
+  outputs = inputs: let
+    eachSystem = inputs.nixpkgs.lib.genAttrs (import inputs.systems);
+    pkgsFor = inputs.nixpkgs.legacyPackages;
+  in {
+    devShells = eachSystem (
+      system: let
+        inherit (pkgsFor.${system}) lib;
```

Comment:
No need to get `lib` from pkgs. Nixpkgs exposes a top-level `lib`, so you can move it to the outer `let in` block.